Manifests/ManifestsApple/com.apple.systemuiserver.plist

```diff
@@ -1559,17 +1564,70 @@
 			<key>pfm_type</key>
 			<string>dictionary</string>
 		</dict>
+		<dict>
+			<key>pfm_description</key>
+			<string>List of menu extras to combine with the user's menu bar</string>
+			<key>pfm_name</key>
+			<string>menuExtras</string>
+			<key>pfm_title</key>
+			<string>Menu Extras</string>
+			<key>pfm_type</key>
+			<string>array</string>
+			<key>pfm_value_unique</key>
+			<true/>
+			<key>pfm_subkeys</key>
```

Comment:
Please sort this alphabetically in the list of keys at the same level, along with all key values within